content/concepts/content-management/03-content-types.md

```diff
@@ -4,18 +4,124 @@ metaTitle: "sensenet - Content Types"
 metaDescription: "sensenet Content Types"
 ---
 
+The content repository contains many different types of content. Content vary in structure and even in function. Different types of content contain different fields and may also implement different business logic. The fields and business logic of a content is defined by its type - the content type.
+
 # Content Types
-https://community.sensenet.com/docs/content-type/
+
+A content type is a special content in the content repository. Content types define the structure and functions of contents. It is defined by a (xml like) configuration file called ``content type definition`` or ``CTD``.
+
+For example a User has a name, e-mail address, etc. - these fields of the user ``content type`` are defined by its ``content type definition``.
+
+> Additional logic, like a specific task when a user is saved can be implemented in the ``content handler``.
 
 # Default content types
+
+In sensenet we provide a default set of predefined content types. These types can be used in any solution built with sensenet but additional (custom) content types can be configured as well according to business needs.
+
+some example from the default set:
+- user
+- folder
+- file
+- workspace
+- image
+- ...
+
+# Custom content type
+
+Since many feature are tied to content types (AllowedChildType, content template) it is strongly suggested to create custom content types according to your solution's architecture. Default content types can be used in your solution but cannot be modified (in SNaaS). You can create custom content type by inheritng from ``GenericContent`` or any other content type in the tree.
```

Comment:
```suggestion
Since many feature are tied to content types (AllowedChildType, content template) it is strongly suggested to create custom content types according to your solution's architecture. Default content types can be used in your solution but cannot be modified or deleted (in SNaaS). You can create custom content type by inheriting from ``GenericContent`` or any other content type in the tree.
```